### PR TITLE
E-307 feat: env

### PR DESCRIPTION
**Pull request by lamartire**

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
-const resolve = require("path").resolve;
-
+import path from "path";
+import fs from "fs";
+import dotenv from "dotenv";
 import cockpit from "./plugins/cockpit.js";
 import endpass from "./plugins/endpass.js";
 
+const { NODE_ENV = "development" } = process.env;
+const envFilePath =
```
Comment (kazurus) [inline nuxt.config.js]: why do we need this? Can we use such [approach](https://github.com/nuxt/nuxt.js/issues/1789#issuecomment-346594758)? [doc section](https://www.npmjs.com/package/dotenv#usage)

File: package.json [hunk 2]
```diff
@@ -6,15 +6,14 @@
   "private": true,
   "scripts": {
     "dev": "nuxt",
-    "build": "nuxt build",
-    "start": "nuxt start",
-    "generate": "nuxt generate",
+    "build": "NODE_ENV=production nuxt build",
```
Comment (kazurus) [inline package.json]: we also need dev commands for devops. Please, use vault [pattern](https://github.com/machinae/endpass-vault/blob/master/package.json#L9)

File: nuxt.config.js (hunk 1 above)
Comment (lamartire) [reply nuxt.config.js]: We should load different files, but by default nuxt dotenv loading `.env` file, the only solution to load files by path. `dotenv.config()` loads `.env` by default too, it is not tolerate to NODE environment too.

Comment (kazurus) [reply nuxt.config.js]: ok, you are right, but can we change code for more flexibility?
```js
path.join(__dirname, `.env.${NODE_ENV}`)
```

**Pull request merged**